### ISSUE TITLE
maxThreads in content sources seems to look at an incorrect location in json 

### ISSUE THREAD

**Issue by whoisbma** [local-projects/ofxApp#2]
https://github.com/local-projects/ofxApp/blob/e4d1c75c6f395e90a271c7c0a1442301a744b16b/src/ofxApp.cpp#L1357

According to the data model in json:
```
"cmsContent": {
        "liveUpdates": {
          "enabled": true,
          "interval": 5.0,
          "maxThreads": 1,
          "maxConcurrentDownloads": 1
        }
      }
```

Shouldn't that line of code look like this?
`const string maxThreadKey = "Content/JsonSources/" + currentContentID + "/liveUpdates/maxThreads";`

**Comment by armadillu** [comment]
Yep please, feel free to fix! It's probably a non-tested case bc there's little reason to have more than 1 thread doing live updates unless you have a great urgency between updates.
